src/main/java/org/openmainframeproject/cobolcheck/features/testSuiteParser/Keywords.java

```diff
@@ -153,12 +153,12 @@ public class Keywords {
                 new Keyword(Constants.TO_EQUAL_KEYWORD,
                         Arrays.asList(),
                         new HashMap<String, List<String>>() {{
-                            put(Constants.EXPECT_KEYWORD, Arrays.asList(
-                                    Constants.NUMERIC_KEYWORD,
-                                    Constants.ALPHANUMERIC_LITERAL_KEYWORD,
-                                    Constants.NUMERIC_LITERAL_KEYWORD,
-                                    Constants.BOOLEAN_VALUE,
-                                    Constants.COBOL_TOKEN)
+                                put(Constants.EXPECT_KEYWORD, Arrays.asList(
```

Comment:
Could these be moved back to original position, so they are without additional indentation?

Reply:
This is for all the changes in this file, regarding additional indentation.